mat-process-api/V1/Boundary/UpdateProcessDataResponse.cs

```diff
@@ -0,0 +1,22 @@
+using mat_process_api.V1.Domain;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace mat_process_api.V1.Boundary
+{
+    public class UpdateProcessDataResponse
+    {
+        public UpdateProcessDataRequest Request { get; set; }
+        public MatProcessData ProcessData { get; set; }
```

Comment:
I'm wondering whether the name of this field is clear enough 'ProcessData'... as in is it clear enough that this is how the updated process data object looks like. It could be called something among the lines of 'UpdatedDocument' or similar... to make it a bit clearer.

It would be good to have second opinions on this one, maybe it's just me.

Reply:
Good suggestion.